src/batterymonitor.h

```diff
@@ -62,7 +62,7 @@
     // Diagramm: 
     //   (Battery)--- [BATTERY_SHIELD_RESISTANCE] ---(INPUT_BOARD)---  [BATTERY_SHIELD_R2] ---(ESP_INPUT)--- [BATTERY_SHIELD_R1] --- (GND)
```

Comment:
Your diagram is different than this one here.
I don't think we should change anything.
There is not clear definition for R1 and R2 how it have to be.

Reply:
> Wemos D1 Mini has an internal Voltage Divider with R1=220K and R2=100K > this means, 3.3V analogRead input voltage results in 1023.0

Then this line has conflicting information, I agree that it is just naming and doesn't really matter. I just wanted to make it more consistent with documentation that is available. I am completely fine reverting all changes and just adapting the values in this single line above though.